partner_private_information/models/res_partner.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Roger Sans <roger.sans@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    private_phone = fields.Char(string="Internal Phone")
+    private_mobile = fields.Char(string="Internal Mobile")
+    private_email = fields.Char(string="Internal Email")
+    private_notes = fields.Text(string="Internal Notes")
+    private_street = fields.Char(string="Internal Street")
+    private_street2 = fields.Char(string="Internal Street2")
+    private_zip = fields.Char(string="Internal Zip", change_default=True)
+    private_city = fields.Char(string="Internal City")
+    private_state_id = fields.Many2one(
+        "res.country.state",
+        string="Internal State",
+        ondelete="restrict",
+        domain="[('country_id', '=?', country_id)]",
```

Comment:
Shouldn't you use private_country_id instead?

Reply:
Done